junit5/junit5-practicesolutions/src/main/java/practice/task16/DevisenkursProvider.java

```diff
@@ -1,8 +0,0 @@
-package practice.task16;
-
-import java.math.BigDecimal;
-
-public interface DevisenkursProvider {
```

Comment:
The history of "DevisenkursProvider.java" ends here. But it has to ccontinue into "ExchangeRateProvider.java". The problem is, there is more than 50% changed, so that git means, DevisenkursProvider.java is deleted and ExchangeRateProvider.java is new created